ui/apps/platform/src/Containers/Collections/converter.test.ts

```diff
@@ -157,58 +180,6 @@ describe('Collection parser', () => {
 
         expect(isCollectionParseError(parseCollection(collectionResponse))).toBeTruthy();
     });
-
-    it('should correctly handle label key/value splitting on `=` delimiter', () => {
```

Comment:
This is no longer relevant since we are not splitting label keys/values in the converter anymore.